changelog.md

```diff
@@ -1,4 +1,5 @@
 # Future
+- [FIXED] Format MSSQL MERGE Statement foreignKey violations as ForeignKeyConstraintError, [#7024](https://github.com/sequelize/sequelize/pull/7024)
```

Comment:
`Throw MSSQL MERGE Statement foreignKey violations as ForeignKeyConstraintError` and no comma in end